### PR TITLE
Disable resilient.co3 warnings

### PR DESCRIPTION
**Pull request by adi88d**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/22416

## Description
Disable resilient.co3 sdk warning: `Unverified HTTPS requests (cafile=false).`
This warning raised when using resilient sdk insecure, Source:
https://github.com/ibmresilient/resilient-python-api/blob/master/resilient/resilient/co3.py#L82

### CODE REVIEW

File: Integrations/integration-IBM_Resilient_Systems.yml [hunk 1]
```diff
@@ -52,8 +52,11 @@ script:
     import requests
     import time
     import json
+    logging.basicConfig()
```
Comment (Itay4) [inline Integrations/integration-IBM_Resilient_Systems.yml]: do we really need that?

File: Integrations/integration-IBM_Resilient_Systems.yml [hunk 2]
```diff
@@ -52,8 +52,11 @@ script:
...
+
     # disable insecure warnings
     requests.packages.urllib3.disable_warnings()
+    resilient.co3.LOG.disable(logging.ERROR)
```
Comment (Itay4) [inline Integrations/integration-IBM_Resilient_Systems.yml]: add comment why this is needed

**Comment by adi88d** [comment]
> did we test this?

I didnt test it, wait for the customer to see if this solve the problem

**Comment by Itay4** [comment]
Closing this due to changes done in `master` branch.
Please reopen in a new branch.

**Pull request closed without merging**